rvnhelper/imageParser.go

```diff
@@ -0,0 +1,116 @@
+package rvnhelper
```

Comment:
will do.

Reply:
Sorry on the lag for this one, I cannot do that as it will require circular dependencies.

Both rvn and rvn-cli import this file.

```
rvn/libvirt.go:	librvnhelp "github.com/rcgoodfellow/raven/rvnhelper"
rvn-cli/rvn.go:	librvnhelp "github.com/rcgoodfellow/raven/rvnhelper"
```

rvn/libvirt needs it because it needs to know where images are put in the filesystem, for example if it is a local user image, a deter image, or image from a URI.  So it needs to import ParseURL from `rvnhelper`

rvn-cli/rvn.go needs it because all of the work it is doing is on validating URLs, copying images from local storage locations to the correct places in the file system, etc.